src/src/common.cpp

```diff
@@ -58,13 +58,14 @@ bool ExpressLRS_AirRateNeedsUpdate = false;
 connectionState_e connectionState = disconnected;
 connectionState_e connectionStatePrev = disconnected;
 
-#ifndef MY_UID
-//uint8_t UID[6] = {48, 174, 164, 200, 100, 50};
-//uint8_t UID[6] = {180, 230, 45, 152, 126, 65}; //sandro unique ID
-uint8_t UID[6] = {180, 230, 45, 152, 125, 173}; // Wez's unique ID
-#else
-uint8_t UID[6] = {MY_UID};
-#endif
+uint32_t HashedBindingPhrase = HASH(My_Binding_Phrase);
```

Comment:
We currently only use 4 bytes but Ive got no probs changing to support 6 in case we find a use for it.  FYI @wvarty binding branch only sends 4 bytes to the rx.

I like the idea of using python to calc sha.  @AlessandroAU any comment?